openeo-geotrellis/src/test/java/org/openeo/geotrellis/TestOpenEOProcessScriptBuilder.java

```diff
@@ -1232,10 +1231,33 @@ public void testArrayFindNoMatch() {
         ByteArrayTile tile1 = ByteConstantNoDataArrayTile.fill((byte) 5, 4, 4);
 
 
-        Tile emptyResult = transformation.apply(JavaConversions.asScalaBuffer(Arrays.asList(tile0, tile1))).head();
+        Tile emptyResult = transformation.apply(JavaConverters.asScalaBuffer(Arrays.asList(tile0, tile1))).head();
         assertTrue(emptyResult.isNoDataTile());
     }
 
+    @DisplayName("Test array_contains process")
+    @Test
+    public void testArrayContains() {
+        OpenEOProcessScriptBuilder builder = new OpenEOProcessScriptBuilder();
+        Map<String, Object> arguments = Collections.emptyMap();
+
+        builder.expressionStart("array_contains", arguments);
+        builder.argumentStart("data");
+        builder.argumentEnd();
+        builder.constantArgument("value",0);
+        builder.expressionEnd("array_contains",arguments);
+
+        Function1<Seq<Tile>, Seq<Tile>> transformation = builder.generateFunction();
```

Comment:
Can we also put an assert on: builder.resultingDataType()

array_contains returns a boolean, so we expect to get BitCellType there.
This may require a change here:

https://github.com/Open-EO/openeo-geotrellis-extensions/blob/5233412dafbb499aa49b86a85a1400c961b9ee59/openeo-geotrellis/src/main/scala/org/openeo/geotrellis/OpenEOProcessScriptBuilder.scala#L1223